pasqal_cloud/client.py

```diff
@@ -149,6 +150,44 @@ def _request(
             "HTTP Client has encountered an issue it is unable to recover from."
         )
 
+    def _request_with_pagination(
```

Comment:
I picked this method from @Mildophin PR but haven't copied the test yet

Reply:
I'll make the same suggestion to rename this `request_all_pages` then :wink: